src/stories/histoire.css

```diff
@@ -0,0 +1,18 @@
+.controls {
+	padding: 10px;
+}
+
+.controls input,
+.controls select {
+  color: white;
```

Comment:
Somehow prettier did not get to format this file.
Please do it manually until automation is in place for `.css` files.